fe/fe-core/src/main/java/org/apache/doris/http/action/WebBaseAction.java

```diff
@@ -67,12 +67,12 @@
             + "  rel=\"stylesheet\" media=\"screen\"/>"
             + "  <link href=\"/static/css?res=Bootstrap-3.3.7/css/bootstrap-theme.css\" "
             + "  rel=\"stylesheet\" media=\"screen\"/>"
-            + "  <link href=\"/static/css?res=DataTables-1.10.21/css/dataTables.bootstrap.css\" "
+            + "  <link href=\"/static/css?res=DataTables-1.10.23/css/dataTables.bootstrap.css\" "
```

Comment:
the version in var.sh is 1.10.25 not 23

Reply:
on branch-0.14, var.sh is 1.10.23. 
so if we change it as general config will better.

Reply:
branch-0.14  is an old branch, your code will merge to master not branch,  you should follow the latest code

Reply:
close this pr, and push it to master branch, Thanks!